sw/device/tests/meson.build

```diff
@@ -1163,6 +1163,32 @@ sw_tests += {
   }
 }
 
+# PWRMGR all reset test.
```

Comment:
Can you also add a bazel BUILD rules?
SW team is transitioning to bazel tools.
Example: https://github.com/lowRISC/opentitan/blob/master/sw/device/tests/sim_dv/BUILD

Reply:
Yes, I added the bazel BUILD rules. Thanks

Reply:
It's passing